src/gobject.cr

```diff
@@ -7,6 +7,7 @@ end
 macro require_gobject(namespace)
   {% unless GObject::LOADED_NAMESPACES.includes?(namespace) %}
     {% GObject::LOADED_NAMESPACES << namespace %}
-    {{run "gobject/generator/build_namespace", namespace}}
+    {{run "./generator/build_namespace", namespace}}
```

Comment:
How did you run things to reproduce the error?

Reply:
cd crystal-gobject/samples
crystal run gdk_window.cr

Reply:
Ah, I added a symlink in `lib` to `src`, so from the project root it should work without these changes, as it has essentially installed the shard into itself. Can you try `cd crystal-gobject; crystal samples/gdk_window.cr`?

Reply:
I install crystal 0.31.1 and launch samples from root directory - all samples work.